### PR TITLE
chore: UnoMaterialSample Uno 4.1.9

### PR DESCRIPTION
**Pull request by iury-kc**
UnoMaterialSample updated to Uno 4.1.9

### CODE REVIEW

File: UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj [hunk 1]
```diff
@@ -14,12 +14,14 @@
...
-    <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
-    <PackageReference Include="Uno.Material" Version="1.0.0-dev.780" />
-    <PackageReference Include="Uno.UI.Skia.Gtk" Version="3.8.6" />
-    <PackageReference Include="Uno.UI.RemoteControl" Version="3.8.6" Condition="'$(Configuration)'=='Debug'" />
+    <PackageReference Include="Uno.Toolkit.UI" Version="1.3.0" />
+    <PackageReference Include="Uno.Toolkit.UI.Material" Version="1.3.0" />
+    <PackageReference Include="Uno.UI" Version="4.1.9" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
+    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.1.9" />
```
Comment (kazo0) [inline UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj:22]: Looks like you're missing he Uno.Material PackageReference

File: UI/UnoMaterialSample/UnoMaterialSample.Droid/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="UnoMaterialSample" android:versionCode="1" android:versionName="1.0">
-  <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="29" />
-  <application android:label="UnoMaterialSample"></application>
-</manifest>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="UnoMaterialSample.UnoMaterialSample" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
```
Comment (kazo0) [inline UI/UnoMaterialSample/UnoMaterialSample.Droid/Properties/AndroidManifest.xml:2]: Is this package name change needed?

File: UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj (hunk 1 above)
Comment (iury-kc) [reply UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj:22]: I added that now. But didn't have any difference. It runs exactly the same way, with our without. Same appearence, no errors.

File: UI/UnoMaterialSample/UnoMaterialSample.Droid/Properties/AndroidManifest.xml (hunk 2 above)
Comment (iury-kc) [reply UI/UnoMaterialSample/UnoMaterialSample.Droid/Properties/AndroidManifest.xml:2]: It was automatic by the VS. I just changed the target to 31.

File: UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj (hunk 1 above)
Comment (kazo0) [reply UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj]: Oh you know what, my bad. I didn't realize this was the Skia.Gtk head, so you only need the Uno.UI.Skia.Gtk package which was already there.

**Review by jeromelaban — APPROVED** [review]

**Review by kazo0 — APPROVED** [review]

**Pull request merged**